litmus-portal/frontend/src/components/WorkflowStepper/index.tsx

```diff
@@ -152,14 +155,25 @@ const CustomStepper = () => {
         weights: arr,
       });
     }
+    setValidYaml(false);
```

Comment:
First Check, If weight.length or arr.length is 0 or not.
If 0, then don't move forward and show error.
If not 0, then move forward and don't show error.